cloudflare/data_source_zone.go

```diff
@@ -0,0 +1,94 @@
+package cloudflare
+
+import (
+	"fmt"
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"log"
+	"regexp"
+)
+
+func dataSourceCloudflareZone() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceCloudflareZoneRead,
+
+		Schema: map[string]*schema.Schema{
+			"zone": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringMatch(regexp.MustCompile("^([a-zA-Z0-9][\\-a-zA-Z0-9]*\\.)+[\\-a-zA-Z0-9]{2,20}$"), ""),
+			},
+			"jump_start": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+			"paused": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			"vanity_name_servers": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"meta": {
```

Comment:
How did you determine where to draw the line with the fields here? I noticed there are a few missing from our enterprise accounts and was curious if it was intended. Happy to provide an anonymised output if you don't have visibility into the full listing.

Reply:
I used the attributes from the `resource_cloudflare_zone`, these look reasonable for a first attempt